frontend/components/text-classifier/results/classifier-annotation/ClassifierAnnotationArea.vue

```diff
@@ -93,6 +93,11 @@ export default {
         this.selectedLabels = this.appliedLabels;
       }
     },
+    allowToShowAllLabels(n, o) {
+      if (n !== o) {
+        this.shownLabels = this.maxVisibleLabels;
+      }
```

Comment:
n and o are not enought explicit, it's better to put words

Reply:
I have removed the watcher, we don't need it